### PR TITLE
Add events polyfill

### PR DESCRIPTION
**Pull request by mocca102**
Copy [MIT licenced](https://github.com/browserify/events) events module for non node environments

### CODE REVIEW

File: src/modules/events.js [hunk 1]
```diff
@@ -0,0 +1,499 @@
+// Copyright Joyent, Inc. and other Node contributors.
+//
```
Comment (esezen) [inline src/modules/events.js:2]: Can we add a link to GH repo?

**Review by esezen — COMMENTED** [review]
This is looking good. I added 2 small comments

File: src/modules/events.js [hunk 2]
```diff
@@ -0,0 +1,499 @@
+// Copyright Joyent, Inc. and other Node contributors.
```
[the remark this answers is not in the document]
Comment (sblaurock) [reply src/modules/events.js:1]: Yes, I think so. Right now I see the concept of "modules" as Constructor products or functionalities. This wouldn't fit that

**Comment by sblaurock** [comment]
We may need to include the Joyent copyright in our source output, but I'll need to research if that's true (feel free to do the same).

**Review by sblaurock — APPROVED** [review]
great work - thank you!

**Pull request merged**